home/models.py

```diff
@@ -208,14 +210,16 @@ def __str__(self) -> str:
 
 
 
-
+#Contact Model
 class Contact(models.Model):
-    name=models.CharField(max_length=100)
-    email=models.CharField(max_length=200)
-    message=models.TextField(max_length=1000)
-    
-    def __str__(self):
-        return self.name
+    name = models.CharField(max_length=100)
+    email = models.EmailField(max_length=200)
+    message = models.TextField(max_length=1000)
+
+    def save(self, *args, **kwargs):
+        self.name = nh3.clean(self.name, tags=[], attributes=[], styles=[], link_rel=[])
+        self.message = nh3.clean(self.message, tags=[], attributes=[], styles=[], link_rel=[])
```

Comment:
![image](https://github.com/user-attachments/assets/68469e39-8a90-4945-b981-b0b3bdb1472b)
please fix this error